src/oadp_resources/oadp/datamover/volume_snapshot_restore.py

```diff
@@ -1,6 +1,49 @@
 from ocp_resources.resource import NamespacedResource
+
+from oadp_constants.oadp.datamover.volume_snapshot_restore import VolumeSnapshotRestorePhase
 from oadp_constants.resources import ApiGroups
 
 
-class VolumeSnapshotLocation(NamespacedResource):
+class VolumeSnapshotRestore(NamespacedResource):
     api_group = ApiGroups.DATAMOVER_OADP_API_GROUP.value
+
+    def is_vsr_done(self):
+        """
+        Check is VSR process is done
```

Comment:
Can you please make sure we use the same documentation format?

Reply:
wdym?